README.md

```diff
@@ -14,18 +14,63 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+## How-to-install
+### Instal for Ubuntu based distro (Ubuntu 16.04)
+It is recommended to install packages with APT (change stable to `master` or `rc` if needed):
+    
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial stable"
+    sudo apt-get update
+    sudo apt-get install -y libindy
+    
+### Instal for Windows
+1. follow to https://repo.sovrin.org/windows/libindy.
+2. download last version of libindy and indy-sdk-deps.
+3. unzip archives to directories, where you want to save working libraries.
+4. add to PATH enviroment variable path to that directories.
+
+### Instal for MacOS        
+Now we haven't prebuild library in some shared place. You can build
+library yourself. Please refer to How-to-build section. 
+After build add to PATH enviroment variable path to builded library.
+            
+### Instal for RHEL based distro (Amazon Linux 2017.03)           
+Now we haven't prebuild library in some shared place. You can build
+library yourself. Please refer to How-to-build section.
+After build add to PATH enviroment variable path to builded library.
+
+## How-to-build
 
 * [Ubuntu based distro (Ubuntu 16.04)](doc/ubuntu-build.md)
 * [RHEL based distro (Amazon Linux 2017.03)](doc/rhel-build.md)
 * [Windows](doc/windows-build.md)
 * [MacOS](doc/mac-build.md)
 
+## How to start local nodes pool with docker
+
+Start local nodes pool on `127.0.0.1:9701-9708` with Docker:
+     
+     ```     
+     docker build -f ci/indy-pool.dockerfile -t indy_pool .
+     docker run -itd -p 9701-9708:9701-9708 indy_pool
+     ```     
+     
+ Ubuntu and Amazon Linux have another approach. Dockerfile `ci/indy-pool.dockerfile` 
```

Comment:
I believe port mapping will work also. Also creation of custom network works on all platforms, but if interface is placed inside of VM it can be hard to access pool by this interface. We need to describe this motivation in a more clean way,